server.js

```diff
@@ -6,7 +6,7 @@ var express = require('express'),
     passport = require('passport'),
     logger = require('mean-logger'),
     io = require('socket.io');
-
+require('dotenv').config();
```

Comment:
Nitpick: Please move this to the line just before the set of `require` statements assigned to variables.